repository/repository.go

```diff
@@ -0,0 +1,69 @@
+package repository
+
+import (
+    "log"
+	"io"
+	"encoding/csv"
+	"os"
+	
+	"github.com/fatih/structs"
+	"github.com/javargas/academy-go-q12021/entities"
+)
+const pathFile = "./DataJobs.csv"
+
+func LoadData() []entities.Job {
+
+	var jobList  []entities.Job = nil
```

Comment:
```suggestion
	jobList := []entities.Job{}
```

Another way to initialize this slice. 😄